newspack-theme/inc/template-functions.php

```diff
@@ -213,12 +213,17 @@ function newspack_body_classes( $classes ) {
 		$classes[] = 'feature-latest';
 	}
 
-	// Add a class when there's an ad background color.
+	// Add a class when there's an ad background color, and another when there's an ad above the footer to remove the space.
 	$ads_background_color = get_theme_mod( 'ads_color', 'default' );
+	$above_footer_ad = get_option( '_newspack_ads_placement_global_above_footer' );
 	if ( 'custom' === $ads_background_color ) {
 		$classes[] = 'ads-bg';
-	}
 
+		if ( str_contains( $above_footer_ad, '"enabled":true' ) ) {
```

Comment:
```suggestion
		if ( true === $above_footer_ad ) {
```

The suggested method above returns a boolean value.